.scalafmt.conf

```diff
@@ -20,6 +20,7 @@ project.excludeFilters = [
   "metals-bench/src/main/resources"
   "tests/unit/src/test/resources"
   "tests/input3/src/main/scala"
+  "tests/input/src/main/scala"
```

Comment:
The line about with `input3` might be removed

Reply:
Done, thanks!